src/shared/components/mutationTable/MutationTable.tsx

```diff
@@ -393,16 +393,14 @@ export default class MutationTable<P extends IMutationTableProps> extends React.
                 MutationTypeColumnFormatter.getDisplayValue(d).toUpperCase().indexOf(filterStringUpper) > -1
         };
 
-        if (this.props.enableGenomeNexus) {
```

Comment:
It is okay to add the column definition in any case, we exclude the column by using the `shouldExclude` property